packages/fxa-auth-server/lib/senders/select_email_services.js

```diff
@@ -94,115 +163,60 @@ module.exports = (log, config, mailer, emailService) => {
     }
 
     if (liveConfig) {
-      try {
-        liveConfig = JSON.parse(liveConfig);
-      } catch (err) {
-        log.error('emailConfig.parse.error', { err: err.message });
+      if (liveConfig === lastSeenLiveConfig) {
+        liveConfig = lastSeenLiveConfigParsed;
+      } else {
+        try {
+          lastSeenLiveConfig = liveConfig;
+          liveConfig = LiveConfig.parse(liveConfig);
+          lastSeenLiveConfigParsed = liveConfig;
+        } catch (err) {
+          lastSeenLiveConfig = lastSeenLiveConfigParsed = null;
+          log.error('emailConfig.parse.error', { err: err.message });
+        }
```

Comment:
The new sandboxing approach seems uniformly faster than the old one in our tests, so there's technically no need for this extra caching logic here. But it just seems like an easy win for a value that doesn't change very often and can be safely cached in memory.

(Also, creating many `SandboxedRegExp` instances at once will allocate lots of memory, which the garbage-collector will probably stay on top of, but still...we can easily save it some work here).